prow/cluster/mem-limit-range_limitrange.yaml

```diff
@@ -1,10 +0,0 @@
-apiVersion: v1
-kind: LimitRange
```

Comment:
Why do we remove this?

Reply:
Because it is yaml for the worker cluster/ is not deployed on the prow cluster. Maybe I should place it in a separate dir?

Reply:
No, that's my bad - and I see that the workers cluster anyway does not have any limitrange in test-pods namespace, so should be fine to remove